lib/DatabaseStatistics.php

```diff
@@ -88,9 +91,9 @@ protected function databaseVersion() {
 	 * @copyright (c) phpBB Limited <https://www.phpbb.com>
 	 * @license GNU General Public License, version 2 (GPL-2.0)
 	 *
-	 * @return int|string
+	 * @return string
 	 */
-	protected function databaseSize() {
+	protected function databaseSize(): string {
```

Comment:
The method might return `$database_size`  as an int  currently. Probably cast it in the return statement.

Reply:
fixed